purchase_order_analytic_search/views/purchase_order_view.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+#---------------------------------------------------------------------------------------------------------
+#   Add analytic account id to purchase orders
+#---------------------------------------------------------------------------------------------------------
+
+   <record id="view_purchase_order_filter_analytic" model="ir.ui.view">
+        <field name="name">purchase.order.list.select</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.view_purchase_order_filter"/>
+        <field name="arch" type="xml">
+            <xpath expr="//search/field[@name='product_id']" position="after">
+                <field name="account_analytic_ids" groups="analytic.group_analytic_accounting"/>
+            </xpath>
+        </field>
+   </record>
```

Comment:
Maybe it could be useful to display the `account_analytic_ids` on tree/list view.